.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,25 @@
+## Expected behavior
+
+## Actual behavior
+
+## Reproducing the behavior
+
+### Computer that is creating the backups
+
+* Operating system, e.g. `Ubuntu Server 14.04.5`:
```

Comment:
Maybe this format is better?

`* Operating system: e.g. Ubuntu Server 14.04.5`